2 data.sql

```diff
@@ -0,0 +1,19 @@
+use MusicNoteDB
+go 
+
+delete MusicRental
+
+insert MusicRental(CustomerFirstName, CustomerLastName, Instrument, Category, MonthlyRentalFee, DateRented, DateReturned)
+select 'Esther', 'Shields', 'Flute', 'Wind', 30, 'Jan 01 2021', 'May 03, 2021'
```

Comment:
Improvement: It would be better if dates were in 'yyyy-mm-dd' format

Reply:
Date format changed to the suggested format